contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -717,3 +717,73 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+/*
+* COLUMN PRIVILEGES
+*/
+
+CREATE OR REPLACE VIEW information_schema_tsql.column_privileges
+AS SELECT CAST(u_grantor.rolname AS sys.nvarchar(128)) AS "GRANTOR",
```

Comment:
GRANTOR pg_roles.rolname is showing the physical PG role name, but we want the logical role name (i.e. the mapped name for TSQL) here, which means `orig_username` stored in `sys.babelfish_authid_user_ext`.
The same applies to GRANTEE and TABLE_SCHEMA nspname. The latter should be selected from `sys.babelfish_namespace_ext.orig_name`

Reply:
Will be updated in next commit.

Reply:
Resolved in latest commit.